p/scripts/main.js

```diff
@@ -1215,6 +1215,84 @@ function init_notifications() {
 }
 // </notification>
 
+// <popup>
+var popup = null,
+	popup_iframe = null,
+	popup_p = null,
+	popup_working = false;
+
+function openPopupWithMessage(msg) {
+	if (popup_working === true) {
+		return false;
+	}
+
+	popup_working = true;
+
+	popup_p.innerHTML = msg;
+	popup_p.removeAttribute('hidden');
+
+	popup.style.display = "block";
+}
+
+function openPopupWithSource(source) {
+	if (popup_working === true) {
+		return false;
+	}
+
+	popup_working = true;
+
+	popup_iframe.src = source;
+	popup_iframe.removeAttribute('hidden');
+
+	popup.style.display = "block";
+}
+
+function closePopup() {
+	popup.style.display = "none";
+
+	popup_iframe.src = 'about:blank';
+	popup_iframe.setAttribute('hidden', true);
+
+	popup_p.setAttribute('hidden', true);
+
+	popup_working = false;
+}
+
+function init_popup() {
+	// Fetch elements.
+	popup = document.getElementById('popup');
+	popup_iframe = document.getElementById('popup-iframe');
+	popup_p = document.getElementById('popup-p');
+
+	//Configure clode-button.
```

Comment:
```suggestion
	//Configure close button.
```

As an aside, I'm not sure if we have a agreed upon standard for this, but I find it much nicer to write `// Comment.`. (I wasn't going to say anything about it since I'm already completely failing to focus on substantial code logic but then you wrote `// Fetch elements.` :-P)

Reply:
Yes, It's my abit to use a space, but I saw that most common doesn't have, so I tried to follow current code-style. I failed on some places :/ ;)